data_portal_api.yml

```diff
@@ -198,17 +332,28 @@ paths:
         A schema that contains a field dictating the visibility of the deployments that will either make the project
```

Comment:
This doesn't seem correct 😆  Looks like in the API design doc, I just forgot to write a description for this API, but this is a description of something else.